src/Integration/LocalServices/IProjectSystemHelperExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IProjectSystemHelperExtensions.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.Shell.Interop;
+using System;
+using System.Linq;
+
+namespace SonarLint.VisualStudio.Integration
+{
+    internal static class IProjectSystemHelperExtensions
+    {
+        /// <summary>
+        /// Returns whether or not a project is of a known test project type.
```

Comment:
It would be clearer if you just said MSTest project type, because VS nowadays recognizes NUnit and XUnit out of the  box (I believe), but this helper does not cover them